### DISCUSSION TITLE
New developer-focused chat room

### DISCUSSION THREAD

**Discussion by iSoron** [iSoron/uhabits#663] [Announcements]
We now have a [**chat room on Gitter**](https://gitter.im/loophabits/dev) for the developers to discuss pull requests and anything else related to coding. If you are a developer and you are planning to contribute to the project, you are welcome to join us there.

If you are a regular user, please do not post feature requests or support questions in this chat room. Feature requests and support questions should be posted to [**GitHub Discussions**](https://github.com/iSoron/uhabits/discussions) instead.

**Comment by hiqua** [comment]
This announcement is no longer visible when one opens the tab "Issues", which I think defeats its purpose.

**Reply by iSoron** [reply]
Sorry, but could you explain why? I am planning to use the Announcements sections for all news about the project, and the Issue tracker strictly for bugs and tasks with clear roadmaps. I hope developers also visit the GitHub Discussions, not only to see these announcements, but also to interact with users.

Following @mh166 suggestion, I have now pinned this discussion to give it a bit more visibility.

**Reply by hiqua** [reply]
I guess it depends if you're fine with new developers opening issues when they struggle with setting up the project. I think for now people are mostly familiar with issues, so they'll open the issue tab and ask their question there instead of here or in the chat room. But it'd be enough to change the bug report template to advertise these other channels anyway, so you can dismiss my comment!

**Reply by iSoron** [reply]
@hiqua Thanks for the suggestion. There is now a link to "Help & FAQ" when creating new issues.

**Comment by mh166** [comment]
I think [pinning this/an issue](https://docs.github.com/en/github/managing-your-work-on-github/pinning-an-issue-to-your-repository) would help (re-)gain visibility. :)

**Comment by mthcorrea** [comment]
Hi i' d like to know How could the app create an automático cloud backup ?

**Reply by iSoron** [reply]
Automatic cloud backups are not currently available.

**Comment by konti-aditi** [comment]
Hello I've been using this app for a while now I love it's features, I have few ideas to enhance the functionality of this app, I'm also an Android developer and would love to contribute here, @iSoron is there any dedicated discord server for the developers of this app Or should I create an issue and discuss the ideas further?

**Reply by hiqua** [reply]
See the gitter link above, but it's not that active fyi. For feature requests you can create new discussions if there is none yet, for bugs you can create issues.
